tests/testthat/test_find_file.R

```diff
@@ -0,0 +1,6 @@
+
+# File set up
+
+context("Find file")
+
+
```

Comment:
This seems to be a stub file. When will tests get added?

Reply:
I cannot get my head around what this function does or what should be tested - It might be helpful if Maelle could have a look at this one with me and help me out! I will ping her.

Reply:
post catch up call - I am resolving this comment as we have a separate issue now open to address this specifically. #48 for reference.